Iterators and Closures/Standard Library Types/Intro/task.md

```diff
@@ -78,6 +78,7 @@ The `Iterator` trait has a number of different methods with default implementati
 Methods that call `next` are called _consuming adaptors_, because calling them uses up the iterator. One example is the `sum` method, which takes ownership of the iterator and iterates through the items by repeatedly calling `next`, thus consuming the iterator. As it iterates through, it adds each item to a running total and returns the total when iteration is complete. The code snippet below has a test illustrating a use of the `sum` method:
```

Comment:
Methods that call `next` are called _consuming adaptors_ because calling them uses up the iterator. One example is the `sum` method, which takes ownership of the iterator and iterates through the items by repeatedly calling `next`, thus consuming the iterator. As it iterates through, it adds each item to a running total and returns the total when the iteration is complete. The code snippet below has a test illustrating the use of the `sum` method: